packages/ag-charts-community/src/module/__snapshots__/optionsModule.test.ts.snap

```diff
@@ -466,9 +466,9 @@ exports[`ChartOptions #prepareOptions for ADV_CHART_CUSTOMISATION it should prep
 exports[`ChartOptions #prepareOptions for ADV_COMBINATION_SERIES_CHART_EXAMPLE it should prepare options as expected 1`] = `
 {
   "axes": {
-    "x": {
+    "__AXIS_ID_2": {
```

Comment:
The ordering of these axis have changed due to the key renaming, the values for each axis have not changed.